Chapter3/1-ModelLoading/Window.cs

```diff
@@ -0,0 +1,152 @@
+﻿using LearnOpenTK.Common;
+using OpenTK.Graphics.OpenGL4;
+using OpenTK.Mathematics;
+using OpenTK.Windowing.Common;
+using OpenTK.Windowing.Desktop;
+using OpenTK.Windowing.GraphicsLibraryFramework;
+
+namespace LearnOpenTK
+{
+    // In this tutorial we focus on importing a model using Assimp and then drawing the model in our scene.
+    public class Window : GameWindow
+    {
+        private Model _backPackModel;
+
+        private Shader _backPackShader;
+
+        private Texture _backPackTexture;
+
+        private Camera _camera;
+
+        private bool _firstMove = true;
+
+        private Vector2 _lastPos;
+
+        public Window(GameWindowSettings gameWindowSettings, NativeWindowSettings nativeWindowSettings)
+            : base(gameWindowSettings, nativeWindowSettings)
+        {
+        }
+
+        protected override void OnLoad()
+        {
+            base.OnLoad();
+
+            GL.ClearColor(0.2f, 0.3f, 0.3f, 1.0f);
+
+            GL.Enable(EnableCap.DepthTest);
+
+            // Backpack by Berk Gedik: https://sketchfab.com/3d-models/survival-guitar-backpack-low-poly-799f8c4511f84fab8c3f12887f7e6b36
```

Comment:
According to the license the model is under you need to not only give attribution to the creator but also link the license.
You also need to specify if any changes where made to the work.

I would suggest creating a `license.txt` or `attribution.txt` in `Resources/Backpack` where you put the sketchfab link, the license link and specify if any changes where made.